pkg/wasm/string.go

```diff
@@ -0,0 +1,45 @@
+//go:build tinygo
+
+package main
+
+import (
+	"strings"
+	"time"
+	"unsafe"
+)
+
+const minSleepDur = time.Millisecond
```

Comment:
I dont think this belongs in `string.go` - it is used in `main.go` and `imports.go`

Reply:
yes. thank you. Maybe I should have sleepMin() function on the sleepingYielder or maybe forceYield( is the better term?